src/mococrw/ecies.h

```diff
@@ -142,6 +142,17 @@ class ECIESEncryptionCtx {
     std::unique_ptr<Impl> _impl;
 };
 
+/**
+ * @brief Get encoded key length in bytes
+ *
+ * Depending on the key length and the elliptic curve point conversion form the key will take different amount
+ * of bytes when encoded. This function can be used to conveniently deduct the encoded length while deserializing.
+ * @param privKey Private key. This is only used to get the key size of the ECIES instance
```

Comment:
```suggestion
 * @param privKey Private key. This is only used to get the public key size of the ECIES instance
```

Reply:
Why is this specific to ECIES? While I think about it, I get the impression that all of these key transformation methods rather belong into key.h? The conversion methods are actually a member of AsymmetricPublicKey, so I would propose to move this code there as well. Will be part of the PrivateKey/Keypair implicitly as well as they inherit from public key.

Reply:
I moved it to AsymmetricPublicKey